evaluation.py

```diff
@@ -0,0 +1,320 @@
+import argparse
+import json
+import os
+import re
+import sys
+from multiprocessing import Pool
+
+import language_evaluation
+import numpy as np
+
+sys.path.append(".")
+from evaluation.gpt_eval import GPTEvaluation
+
+
+class evaluation_suit:
```

Comment:
Please cite the original authors and add a link to the repo where the code is from. For complete attribution, we should either add the license information at the top of the file here as well or add the original license to the folder.